### PR TITLE
chore: Update reference to changelog in PR template

### PR DESCRIPTION
**Pull request by tomkerkhove**
## Changes

Update reference to changelog in PR template since the current one is not correct.

### CODE REVIEW

**Review by pellared — CHANGES_REQUESTED** [review]
What do you mean by "it does not exist"?

See:

image

**Review by reyang — CHANGES_REQUESTED** [review]
This is not correct.
There are many `CHANGELOG.md` files in the project - each package has its own.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -6,6 +6,6 @@ Please provide a brief description of the changes here.
 
 For significant contributions please make sure you have completed the following items:
 
-* [ ] `CHANGELOG.md` updated for non-trivial changes
+* [ ] `src/OpenTelemetry/CHANGELOG.md` updated for non-trivial changes
```
Comment (pellared) [inline .github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion
* [ ] Appropriate `CHANGELOG.md` updated for non-trivial changes
```

Comment (tomkerkhove) [reply .github/PULL_REQUEST_TEMPLATE.md]: Good suggestion, thanks!

**Review by pellared — APPROVED** [review]

**Review by reyang — APPROVED** [review]

**Pull request merged**